content/en/guide/v10/components.md

```diff
@@ -87,10 +87,10 @@ In order to have the clock's time update every second, we need to know when `<Cl
 | `componentWillMount()`        | (deprecated) before the component gets mounted to the DOM
 | `componentDidMount()`         | after the component gets mounted to the DOM
 | `componentWillUnmount()`      | prior to removal from the DOM
-| `componentWillReceiveProps(nextProps, nextState)` | before new props get accepted _(deprecated)_
-| `getDerivedStateFromProps(nextProps)` | just before `shouldComponentUpdate`. Use with care.
-| `shouldComponentUpdate(nextProps, nextState)` | before `render()`. Return `false` to skip render
-| `componentWillUpdate(nextProps, nextState)` | before `render()` _(deprecated)_
+| `componentWillReceiveProps(nextProps, context)` | before new props get accepted _(deprecated)_
+| `getDerivedStateFromProps(nextProps, prevState)` | just before `shouldComponentUpdate`. Use with care.
+| `shouldComponentUpdate(nextProps, nextState, context)` | before `render()`. Return `false` to skip render
+| `componentWillUpdate(nextProps, nextState, context)` | before `render()` _(deprecated)_
```

Comment:
```suggestion
| `componentWillReceiveProps(nextProps, nextContext)` | before new props get accepted _(deprecated)_
| `getDerivedStateFromProps(nextProps, prevState)` | just before `shouldComponentUpdate`. Use with care.
| `shouldComponentUpdate(nextProps, nextState, nextContext)` | before `render()`. Return `false` to skip render
| `componentWillUpdate(nextProps, nextState, nextContext)` | before `render()` _(deprecated)_
```

Reply:
Only just now saw your comment re:nextContext, sorry!

I think consistency is better as just `context` creates ambiguity alongside other `nextX` props.